operatorconfig/moduleconfig/authorization/v1.10.0/deployment.yaml

```diff
@@ -302,91 +381,181 @@ spec:
     targetPort: 50051
     name: grpc
 ---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: storage-service-selfsigned
+  namespace: <NAMESPACE>
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: storage-service-selfsigned
+  namespace: <NAMESPACE>
+spec:
+  secretName: storage-service-selfsigned-tls
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  subject:
+    organizations:
+    - dellemc
+  isCA: false
+  privateKey:
+    algorithm: RSA
+    encoding: PKCS1
+    size: 2048
+  usages:
+    - client auth
+  dnsNames:
+  - csm-authorization-storage-service
+  issuerRef:
+    name: storage-service-selfsigned
+    kind: Issuer
+    group: cert-manager.io
+---
 # Redis
-apiVersion: apps/v1 
-kind: Deployment
+apiVersion: v1
+kind: Service
 metadata:
-  name: redis-primary
+  name: <AUTHORIZATION_REDIS_NAME>
   namespace: <NAMESPACE>
-  labels:
-    app: redis
 spec:
+  type:
+  clusterIP: None
+  selector:
+    app: <AUTHORIZATION_REDIS_NAME>
+  ports:
+  - protocol: TCP
+    port: 6379
+    targetPort: 6379
+    name: <AUTHORIZATION_REDIS_NAME>
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: <AUTHORIZATION_REDIS_NAME>
+  namespace: <NAMESPACE>
+spec:
+  serviceName: <AUTHORIZATION_REDIS_NAME>
+  replicas: 5
   selector:
     matchLabels:
-      app: redis
-      role: primary
-      tier: backend
-  replicas: 1
+      app: <AUTHORIZATION_REDIS_NAME>
   template:
     metadata:
       labels:
         csm: <NAME>
-        app: redis
-        role: primary
-        tier: backend
+        app: <AUTHORIZATION_REDIS_NAME>
     spec:
+      initContainers:
+      - name: config
+        image: <AUTHORIZATION_REDIS_IMAGE>
+        env:
+          - name: REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: redis-csm-secret
+                key: password
+
+        command: [ "sh", "-c" ]
+        args:
+          - |
+            cp /csm-auth-redis-cm/redis.conf /etc/redis/redis.conf
+              
+            #REDIS_PASSWORD=`echo $REDIS_PASSWORD | base64 -d`
```

Comment:
remove the commented line as well as the next line, which was used to display password for debugging.